### PR TITLE
flattened if else statements for better code structure

### PR DESCRIPTION
**Pull request by jaykay99992**
changed 
```

if:

else:
    if:

    else:
```

to 

```

if:

elif:

else:
```

for better code structure and clarity.

### CODE REVIEW

File: src/flask/cli.py [hunk 1]
```diff
@@ -298,22 +298,20 @@ def load_app(self) -> Flask:
         if self._loaded_app is not None:
             return self._loaded_app
 
-        if self.create_app is not None:
+    if self.create_app is not None:
```
Comment (JeremieBergeron) [inline src/flask/cli.py]: Correct the code indentation. The code doesn't run. Please, read our contributing guide: https://github.com/jaykay9999/flaskreviewthree/blob/main/CONTRIBUTING.rst#submitting-patches

**Review by JeremieBergeron — CHANGES_REQUESTED** [review]

File: src/flask/cli.py (hunk 1 above)
Comment (jaykay99992) [reply src/flask/cli.py]: @JeremieBergeron just fixed code indentation, thank you for the remark, should I merge it like this ?

Comment (JeremieBergeron) [reply src/flask/cli.py]: @jaykay9999 Could you say if we need to had an entry in ``CHANGES.rst`` for this PR like said in the [contributing guide](https://github.com/jaykay9999/flaskreviewthree/blob/main/CONTRIBUTING.rst#submitting-patches)?
The docs says we should, but it seems overkill add entry for a small refactor like this one.

Comment (jaykay99992) [reply src/flask/cli.py]: @JeremieBergeron you're correct, for a change have an entry to CHANGES.rst  we must also include .. versionchanged:: which might be too much for a small change. Do I proceed to merge the code like this ?

Comment (JeremieBergeron) [reply src/flask/cli.py]: Yes, you can merge.

**Review by JeremieBergeron — APPROVED** [review]

**Comment by jaykay9999** [comment]
Code approved and merged

**Pull request merged**